helix-core/src/main/java/org/apache/helix/util/RebalanceUtil.java

```diff
@@ -136,4 +141,13 @@
     result[1] = slaveStateValue;
     return result;
   }
+
+  public static void scheduleInstantPipeline(String clusterName) {
+    GenericHelixController controller = GenericHelixController.getController(clusterName);
+    if (controller != null) {
+      controller.scheduleInstantRebalance();
+    } else {
+      LOG.warn("Failed to issue a pipeline run for cluster {}.", clusterName);
```

Comment:
Can you make the log message clear? Under what circumstances would controller be null?

Reply:
Done.